requirements.txt

```diff
@@ -79,4 +79,5 @@ djangorestframework-guardian==0.3.0
 flake8==3.8.2 
 black==19.10b0
 quark-engine==20.8
-git+git://github.com/DissectMalware/XLMMacroDeobfuscator.git@c78b9e443e2667399470462fd863ebe4e2b8c978
\ No newline at end of file
+git+git://github.com/DissectMalware/XLMMacroDeobfuscator.git@c78b9e443e2667399470462fd863ebe4e2b8c978
+git+git://github.com/fireeye/speakeasy.git@e5f7bcf0f1ddf10614f32e76b5d322a5387185db
```

Comment:
Is there a particular reason you used the GitHub url ? [Speakeasy is available on PyPi](https://pypi.org/project/speakeasy-emulator/) and it would be better to grab it from there.

Reply:
To be honest I didn't know it was provided in pypi. If you think is a better solution, will make this change.